src/os/ui/ol/imap.js

```diff
@@ -31,21 +32,23 @@ os.ui.ol.IMap.prototype.addFeature;
 /**
  * Adds an array of features to the drawing layer.
  * @param {!Array<!ol.Feature>} features The features to add
+ * @param {Object=} opt_style Optional feature style to apply to all features
+ * @param {ol.layer.Layer=} opt_layer Optional layer to which to add the features
  * @return {!Array<!ol.Feature>}
  */
 os.ui.ol.IMap.prototype.addFeatures;
 
 
 /**
- * Removes a feature from the drawing layer
```

Comment:
There is still a little variation between this interface and the implementations in `os.MapContainer` and `os.ui.ol.OLMap`. All of these functions were originally intended to abstract the drawing layer from code that needed to manage temporary features on the map.

It looks like the `opt_layer` arguments were added to reuse these functions for the search layer when that was split out from the drawing layer. I'm not very fond of that personally, and I'd prefer if `os.ui.search.FeatureResultCardCtrl` managed features directly on the search layer/source instead of routing that through the map container.